python/ray/_private/node.py

```diff
@@ -1232,17 +1234,12 @@ def start_head_processes(self):
 
         if self._ray_params.include_dashboard is None:
             # Default
-            include_dashboard = True
-            raise_on_api_server_failure = False
-        elif self._ray_params.include_dashboard is False:
-            include_dashboard = False
             raise_on_api_server_failure = False
         else:
-            include_dashboard = True
-            raise_on_api_server_failure = True
+            raise_on_api_server_failure = self._ray_params.include_dashboard
```

Comment:
nit: this flag seems to be an no-op now? technically we don't need it anymore? 

i believe it's the same as `incldue_dashboard`

Reply:
not quite; if `include_dashboard` is None, we want this to be `False`